lib/client/index.js

```diff
@@ -36,7 +39,34 @@ module.exports = ({ port = null, config = {}, filters = {} }) => {
       version,
     };
 
-    return res.status(status).json(data);
+    // internal validation URL is optional, skip it if it doesn't exist
+    if (!config.brokerClientValidationUrl) {
+      return res.status(status).json(data);
+    }
+
+    // add the internal validation URL to the response data and check it
+    data.brokerClientValidationUrl = logger.sanitise(config.brokerClientValidationUrl);
+    request(config.brokerClientValidationUrl, { timeout: 5000 }, (error, response) => {
+      if (error) {
+        data.brokerClientValidationUrlSuccess = false;
+        data.error = error.message;
+        data.ok = false;
+        return res.status(500).json(data);
```

Comment:
Do we want to know about `isConnOpen` here? I don't see it being added to `data`, so this potentially important info will not be in the response.

Reply:
It appears on the `websocketConnectionOpen` key once `data` is declared.

The code is slightly convoluted as I wanted to stick to the callback-way of calling `request` like in the rest of the codebase. Open to suggestions on how to improve, what do you think would be easier to grok?

Reply:
Once I uncollapsed the diff on GH I spotted the declaration! I think the confusing part is actually that `status` refers specifically to the result of `isConnOpen`.

What do you think about calling it `isConnOpenStatus`, or query `isConnOpen` boolean at the point we want to determine the status code?

Reply:
Naming is hard! `isX` is a predicate, so `isConnOpenStatus` doesn't sound proper to me. Maybe `isConnStatusOpen`?

The split flow here is to accommodate the optionality of the validation url, and use `request` in a callback fashion if it is present. Sort of older-school JS... Didn't want to add promise-based `request` lib or similar, or introduce plain `Promise`s just for this flow.